### PR TITLE
fix: 修复空页面执行右键删除和右键复制时控制台报错的问题

### PR DESCRIPTION
**Pull request by gargameljyh**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/canvas/src/components/container/CanvasMenu.vue [hunk 1]
```diff
@@ -108,10 +108,16 @@ export default {
 
     const operations = {
       del() {
-        removeNodeById(getCurrent().schema.id)
+        const currentSchemaId = getCurrent().schema?.id
+        if (currentSchemaId) {
+          removeNodeById(currentSchemaId)
```
Comment (ghost) [inline packages/canvas/src/components/container/CanvasMenu.vue]: removeNodeById函数中有对传入的id做判空处理，确定是否是该问题导致的

Comment (gargameljyh) [reply packages/canvas/src/components/container/CanvasMenu.vue]: 问题是 getCurrent().schema为空，这里主要是加?.断言判断

File: packages/canvas/src/components/container/CanvasMenu.vue [hunk 2]
```diff
@@ -111,10 +111,10 @@ export default {
 
     const operations = {
       del() {
-        removeNodeById(getCurrent().schema.id)
+        removeNodeById(getCurrent().schema?.id)
```
Comment (hexqi) [inline packages/canvas/src/components/container/CanvasMenu.vue:118]: 这类问题建议优先界面置灰或隐藏，修改代码添加容错只是解决了报错问题，虽然不报错但用户点击后仍然是没有任何反馈。

**Review by hexqi — COMMENTED** [review]
建议继续优化下

File: packages/canvas/src/components/container/CanvasMenu.vue [hunk 3]
```diff
@@ -163,7 +176,7 @@ export default {
       }
 
       if (item?.code) {
-        operations[item.code](item)
+        operations[item.code]?.(item)
```
Comment (hexqi) [inline packages/canvas/src/components/container/CanvasMenu.vue:179]: disabled的情况下直接在前面return感觉更好，当前虽然不会执行operations里面的方法，但菜单也被关掉了

Comment (hexqi) [inline packages/canvas/src/components/container/CanvasMenu.vue:179]: disabled的情况下直接在前面return感觉更好，当前虽然不会执行operations里面的方法，但菜单也被关掉了

**Review by hexqi — APPROVED** [review]

**Review by hexqi — APPROVED** [review]

**Review by ghost — APPROVED** [review]
LGTM

**Review by wenmine — APPROVED** [review]

**Pull request merged**